### PR TITLE
Reimplements the old mining station

### PR DESCRIPTION
**Pull request by ArcLumin**
[Changelogs]:
[]: 
:cl: Mining Station Old
add: Added the old mining station. Accessible via science shuttle, labor shuttle, and mining shuttle. Labor shuttle and mining shuttle can still go to lavaland
/:cl:

### CODE REVIEW

**Comment by KayeArray** [comment]
YYYYYYYYYYYYYYYYYYYYYYYYY YEAH YEAH YEAH YEAH YEAH YEAH YEAH YEAH YEAH YEAH

File: _maps/tgstation2.dm [hunk 1]
```diff
@@ -5,16 +5,17 @@
 		#define MINETYPE "lavaland"
 
         #include "map_files\TgStation2\tgstation.2.1.3.dmm"
-        #include "map_files\generic\z2.dmm"
+        #include "map_files\generic\z2 centcom.dmm"
         #include "map_files\generic\z3.dmm"
         #include "map_files\generic\z4.dmm"
-        #include "map_files\generic\lavaland.dmm"
+        #include "map_files\generic\z5 lavaland.dmm"
```
Comment (Carbonhell) [inline _maps/tgstation2.dm:11]: ...why are there two lavalands?

File: _maps/tgstation2.dm [hunk 2]
```diff
@@ -5,16 +5,17 @@
...
         #include "map_files\generic\z6.dmm"
         #include "map_files\generic\z7.dmm"
         #include "map_files\generic\z8.dmm"
 		#include "map_files\generic\z9.dmm"
 		#include "map_files\generic\z10.dmm"
 		#include "map_files\generic\z11.dmm"
```
Comment (Carbonhell) [inline _maps/tgstation2.dm:17]: what's this random indentation..?i don't get it

File: _maps/tgstation2.dm (hunk 1 above)
Comment (ArcLumin) [reply _maps/tgstation2.dm:11]: What do you mean? There isn't

File: _maps/tgstation2.dm (hunk 2 above)
Comment (ArcLumin) [reply _maps/tgstation2.dm:17]: That's how it is in the natural file. I just left it as is

File: _maps/tgstation2.dm (hunk 1 above)
Comment (Carbonhell) [reply _maps/tgstation2.dm:11]: fuck, i see double now, nvm

**Pull request merged**